modules/shell/command.go

```diff
@@ -50,6 +52,35 @@ func RunCommandAndGetOutput(t *testing.T, command Command) string {
 // RunCommandAndGetOutputE runs a shell command and returns its stdout and stderr as a string. The stdout and stderr of that command will also
 // be printed to the stdout and stderr of this Go program to make debugging easier.
 func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
+	allOutput := []string{}
+	err := runCommandAndStoreOutputE(t, command, &allOutput, &allOutput)
+
+	output := strings.Join(allOutput, "\n")
+	return output, err
+}
+
+// RunCommandAndGetStdOut runs a shell command and returns solely its stdout (but not stderr) as a string. The stdout
+// and stderr of that command will also be printed to the stdout and stderr of this Go program to make debugging easier.
```

Comment:
NIT: I've been updating these functions without `E` to also mention that the test will fail if there are any errors in running the command. See `k8s` module for example: https://github.com/gruntwork-io/terratest/blob/master/modules/k8s/cluster_role.go#L11

Reply:
https://github.com/gruntwork-io/terratest/pull/324/commits/4faa99995986c111de3296cd604c3a5421e6ed3b